marlowe-runtime-web/src/Language/Marlowe/Runtime/Web/Types.hs

```diff
@@ -231,8 +232,15 @@ data ContractState = ContractState
   , state :: Maybe Semantics.State
   , utxo :: Maybe TxOutRef
   , txBody :: Maybe TextEnvelope
+  , unclaimedPayouts :: [Payout]
   } deriving (Show, Eq, Generic)
 
+data Payout
+    = Payout
+        { payoutId :: TxOutRef
+        , role :: Text} -- TODO (N.H) : replace it by an asset (policyId,tokenName) vs just a role (~TokenName)
```

Comment:
Actually this `role` type is fine; what we want to do is add an `assets` field which says what funds were paid out.

Reply:
ok ok